src/utils/config.ts

```diff
@@ -210,9 +210,9 @@ const configParsers = {
     },
     GEMINI_MODEL(model?: string) {
         if (!model || model.length === 0) {
-            return 'gemini-1.5-pro-latest';
+            return 'gemini-1.5-flash-latest';
```

Comment:
Please update this code as [pull request](https://github.com/tak-bro/aicommit2/pull/47)

Reply:
Updated!